pkg/ebpf/c/tracee.bpf.c

```diff
@@ -6060,6 +6061,41 @@ int BPF_KPROBE(trace_security_inode_rename)
     return 0;
 }
 
+SEC("kprobe/kallsyms_lookup_name")
+TRACE_ENT_FUNC(kallsyms_lookup_name, KALLSYMS_LOOKUP_NAME);
+
+SEC("kretprobe/kallsyms_lookup_name")
+int BPF_KPROBE(trace_ret_kallsyms_lookup_name)
+{
+    event_data_t data = {};
+    if (!init_event_data(&data, ctx))
+        return 0;
+
+    if (!should_trace(&data))
+        return 0;
+
+    args_t saved_args;
+    if (load_args(&saved_args, KALLSYMS_LOOKUP_NAME) != 0)
+        return 0;
+    del_args(KALLSYMS_LOOKUP_NAME);
+
+    char *name = (char *) saved_args.args[0];
+    unsigned long address = PT_REGS_RC(ctx);
+
+    if (should_submit(KALLSYMS_LOOKUP_NAME, data.config)) {
```

Comment:
@yanivagman I see some inconsistencies between kprobes regarding this line, in some we have this check and in some we don't (for example security_sb_mount). I think we can usually skip this check since kprobes are basically never attached without submitting but WDYT?
Whatever we choose we need to normalize throughout the codebase.

Reply:
I believe we should normalize.. but it does make sense to have this sometimes (as sometimes the probes are enabled but the event shouldn't be sent (specially in my new network code cases)). I do agree we should normalize though, and check all cases (maybe during the eBPF code split/spinoff time ?).